package.json

```diff
@@ -162,5 +162,8 @@
     ],
     "sourceMap": true,
     "instrument": true
+  },
+  "optionalDependencies": {
+    "fsevents": "^2.1.3"
```

Comment:
This was added to get the Github tests for Windows to work a couple months ago. A warning message was causing the tests to timeout on Windows for certain versions of Node (v12 if I recall correctly). There was a similar warning message in the SO question below and the solution was recommended in the comments:

"How to solve npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13"
https://stackoverflow.com/questions/62810078/how-to-solve-npm-warn-optional-skipping-optional-dependency-fsevents1-2-13

The tests on Github appear to have been upgraded once or twice since a couple months ago and NPM v7 apparently addresses this issue (see SO question below) so I'm not sure if it is necessary now. The fix itself in the PR does not need this dependency as far as I know. Also there was no significant difference in bundle size between builds for this PR and the master branch, at least on Mac OS (for the test project the PR version was actually a few hundred bytes smaller).

https://stackoverflow.com/questions/40226745/npm-warn-notsup-skipping-optional-dependency-unsupported-platform-for-fsevents/40464936#40464936

Reply:
Hi Conduitry, I checked and I don't think the `"optionalDependencies": {
    "fsevents": "^2.1.3"` in package.json is needed at all.

I used a VM (VirtualBox running Windows 10) to clone both this fork and the sveltejs:master branch and ran npm install using v12.22.6 (which was one of the versions the Github tests a couple months ago and was timing out). The warning `npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules\fsevents: npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: {"os":"darwin", "arch":"any"} (current: {"os":"win32","arch":x64"})` came up for both versions. All tests passed for both (including the tests I wrote on this fork).

Adding the above to package.json gets rid of the warning. I thought may have been causing the Github tests on Windows using Node v12 to timeout for this fork, but the same warning happens in the sveltejs:master version. In any event, we don't want to add an extra dependency for Windows users if it isn't needed.

TLDR I'll get rid of this, it shouldn't be necessary.